tests/parser/syntax/test_interfaces.py

```diff
@@ -134,6 +134,83 @@ def f(a: uint256): # visibility is nonpayable instead of view
     """,
         InterfaceViolation,
     ),
+    (
+        # `receiver` of `Transfer` event should be indexed
+        """
+from vyper.interfaces import ERC20
+
+implements: ERC20
+
+event Transfer:
+    sender: indexed(address)
+    receiver: address
+    value: uint256
+
+event Approval:
+    owner: indexed(address)
+    spender: indexed(address)
+    value: uint256
+
+name: public(String[32])
+symbol: public(String[32])
+decimals: public(uint8)
+balanceOf: public(HashMap[address, uint256])
+allowance: public(HashMap[address, HashMap[address, uint256]])
+totalSupply: public(uint256)
+
+@external
+def transfer(_to : address, _value : uint256) -> bool:
+    return True
+
+@external
+def transferFrom(_from : address, _to : address, _value : uint256) -> bool:
+    return True
+
+@external
+def approve(_spender : address, _value : uint256) -> bool:
+    return True
+    """,
+        InterfaceViolation,
+    ),
+    (
+        # `receiver` of `Transfer` event should be indexed
+        # instead of `value`
+        """
+from vyper.interfaces import ERC20
+
+implements: ERC20
+
+event Transfer:
+    sender: indexed(address)
+    receiver: address
```

Comment:
let's keep `receiver` indexed -- that way the only thing wrong with this implementation is that `value` is indexed.